### DISCUSSION TITLE
Do you have a version of Vue

### DISCUSSION THREAD

**Discussion by AndriodStudent** [epam/ketcher#465] [General]
Do you have a version of Vue；Please help me！！！

**Comment by AndreiMazol** [comment]
Dear @AndriodStudent ,

this moment Ketcher has only one version of presentation layer that  is based on  React (ketcher-react npm package). We are working on ketcher-core package that will allow to use Ketcher without dependencies on client libs or frameworks. Ready-to-use version of it will be released in few months.

Best regards,
Ketcher team
